src/types.ts

```diff
@@ -1,5 +1,7 @@
 import type { Address } from "./Address"
+import { Ownable } from "./Ownable"
```

Comment:
```suggestion
import type { Ownable } from "./Ownable"
```
Because `Ownable` is used only for type annotations and declarations.